### PR TITLE
[stable/percona]: Resource and image overrides for initcontainer

### PR DESCRIPTION
**Pull request by tokheim**
Signed-off-by: Asmund Tokheim <tokheim@outlook.com>

#### Is this a new chart

No

#### What this PR does / why we need it:

Sets resources for percona init container so it can be used in namespaces with resource quotas. Also allows overriding init container image as most other charts do.

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/chart]`)

### CODE REVIEW

**Comment by costarag** [comment]
:+1:

File: stable/percona/Chart.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: percona
-version: 1.1.0
+version: 1.1.1
```
Comment (davidkarlsen) [inline stable/percona/Chart.yaml]:
```suggestion
version: 1.2.0
```

**Review by davidkarlsen — CHANGES_REQUESTED** [review]

File: stable/percona/Chart.yaml (hunk 1 above)
Comment (tokheim) [reply stable/percona/Chart.yaml]: Thanks @davidkarlsen . I had some trouble with company sso over github ui, so committed the same fix.

**Pull request merged**